### PR TITLE
Added griis() function

### PR DESCRIPTION
**Pull request by ibartomeus**
Let me know what do you think of this implementation. Thanks!

### CODE REVIEW

File: R/griis.R [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+                  "brackish", "host")){
+    stop("type should be one of terrestrial, freshwater,
+         marine, brackish, host")
+    }
+  }
+  #Parse url and extract table
+  if(!is.null(name)){
+    name <- gsub(" ", "%20", name, fixed = TRUE)
+  }
+  url <- paste0(griis_base(), "name=", name, "&impacts=", impacts,
```
Comment (sckott) [inline R/griis.R]: I'd prefer that we use `httr::GET` here to have better control over the http request, proper failure behavior, etc. , and `httr::stop_for_status()` so user gets a hopefully informative error message if the request fails

Comment (sckott) [inline R/griis.R]: And you could simply param arguments here by doing e.g., `orc(list(name = name, impacts = impacts))` etlc. with remainder of arguments, where `orc` is https://github.com/ropenscilabs/originr/blob/master/R/zzz.R#L3

File: R/griis.R [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+                "&verified=", verified, "&country=", country,
+                "&kindom=", kindom, "&type=", type)
+  doc <- read.table(url, header = TRUE, sep = ";")
+  colnames(doc)[7] <- "Evidence.of.Impacts"
```
Comment (sckott) [inline R/griis.R]: (regarding lines 204-206) Is there any chance this number based indexing will fail?  would be more robust if you could index by column name if possible

File: R/griis.R [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+#'
+#' @author Ignasi Bartomeus \email{nacho.bartomeus@@gmail.com}
+#' @examples \dontrun{
+#' griis(name = "Carpobrotus edulis")
+#' griis(name = "Carpobrotus edulis", country = "Portugal")
+#' }
+#'
+griis <- function(name = NULL, impacts = NULL, verified = NULL, country = NULL,
+                  kindom = NULL, type = NULL){
+  #species is well spelled (should we check this with taxize, or people can do it beforehand?)
```
Comment (sckott) [inline R/griis.R]: probably just let people do it separately

File: R/griis.R [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+    }
+  }
+  #Parse url and extract table
+  if(!is.null(name)){
+    name <- gsub(" ", "%20", name, fixed = TRUE)
+  }
+  url <- paste0(griis_base(), "name=", name, "&impacts=", impacts,
+                "&verified=", verified, "&country=", country,
+                "&kindom=", kindom, "&type=", type)
+  doc <- read.table(url, header = TRUE, sep = ";")
```
Comment (sckott) [inline R/griis.R]: `stringsAsFactors = FALSE`

File: R/griis.R (hunk 1 above)
Comment (ibartomeus) [inline R/griis.R]: orc not needed, becasue the url need all args even if null

Comment (sckott) [inline R/griis.R]: ok

**Comment by ibartomeus** [comment]
Not sure if you get a notification when I commit new changes. I did :)

**Comment by sckott** [comment]
oh good, so you're all set now?

**Comment by ibartomeus** [comment]
I think so.

On Sat, Jun 4, 2016 at 5:46 PM Scott Chamberlain notifications@github.com
wrote:

> oh good, so you're all set now?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> https://github.com/ropenscilabs/originr/pull/7#issuecomment-223762554,
> or mute the thread
> https://github.com/notifications/unsubscribe/AB8IgqVwektVpjWeMv1CfWz9REgsa5Qzks5qIZ3GgaJpZM4Im-Ys
> .
> 
> ## 
> 
> ## 
> 
> Ignasi Bartomeus PhD
> www.bartomeuslab.com
> @ibartomeus
> Skype: nachobartomeus

Dpto. Ecología Integrativa
Estación Biológica de Doñana (EBD-CSIC)
Avda. Américo Vespucio s/n
Isla de la Cartuja
41092, Sevilla (Spain)

**Pull request merged**